libraries/api/src/middleware/schema.js

```diff
@@ -45,12 +45,9 @@ const validateSchemas = ({ validator, absoluteSchemas, rootUrl, serviceName, ent
     // If input schema is defined we need to validate the input
     if (input) {
       if (!typeis(req, 'application/json')) {
-        return res.reportError(
-          'MalformedPayload',
-          'Payload must be JSON with content-type: application/json ' +
-          'got content-type: {{contentType}}', {
-            contentType: req.headers['content-type'] || null,
-          });
+        debug(
```

Comment:
As discussed in Slack, we need to log the request URL or at least the task ID in order to hunt down misbehaving clients in production, and `debug` won't add this to the production logs. I'm not familiar with the service code to know what to use, and if we want to log in central logs or send as a message to Sentry